quake1_example/class/_func.gd

```diff
@@ -7,8 +7,16 @@ func _get_properties(dict : Dictionary) : props = dict
 var aabb : AABB
 
 func _gen_aabb() :
-	#aabb.position = global_position
 	for m in get_children() :
 		if m is GeometryInstance3D :
 			aabb = aabb.merge(m.get_aabb())
 	aabb.position += global_position
+
+func _can_create_trigger() -> bool : return false;
+func _create_trigger() -> void : pass
+
+func _touch_horizontal(nor : Vector3) -> bool :
```

Comment:
Just some helper functions can be used to check if player touch sides or top of object